htmltools/_core.py

```diff
@@ -1238,7 +1238,9 @@ def _static_extract_serialized_html_deps(
 # =============================================================================
 # HTML strings
 # =============================================================================
-class HTML(str):
+
+
+class HTML:
```

Comment:
from pairing w/ Barret, If we want to keep the str methods, can inherit from collections.UserString (cf https://github.com/posit-dev/py-shiny/issues/1439)

Reply:
Using!